### PR TITLE
Consolidate metro config files and split tester bundle

### PR DESCRIPTION
**Pull request by JasonVMo**
This changes metro bundling a bit by:

- Create a unified configuration pattern for metro.config.js files out of the scripts directory
- Splitting the tester out of the react-native-uifabric directory, this is to separate the tester vs. implementation packages
- Reworked some of the monorepo tasks to make them parse the lerna config
- Added some utilities for resolving files and modules with absolute paths

the yarn bundle command will bundle across all the various sub-projects.  To bundle in the experiments/tester directory just run yarn bundle in that directory.

Next steps are:

- get the expo metro configs to share the configuration
- move the actual win32 bundling into the apps directory
- use the same process for consolidating jest files as well

### CODE REVIEW

**Comment by JasonVMo** [comment]
I also removed the office-ui-fabric-react dependency in the uifabric package as well as the unused .doc.tsx files.

File: experiments/tester/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+`RNTester` is the test app that we use to test our FluentUI components during development.
+
+## Launch `RNTester` app
+
+1. After you've cloned the repository for the first time, follow the instructions [here](../../README.md) to install npm packages and build the entire repository.
+
+2. Then go into `experiments\react-native-uifabric` folder:
```
Comment (dotnetjunky) [inline experiments/tester/README.md]: I think you need to update folder path here.

File: experiments/tester/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+2. Follow the same step #2 as above.
+3. Start the debug server:
+
+```
+    yarn start:tester
+```
+
+4. Open Chrome and navigate to http://localhost:8081/debugger-ui
+
+5. Open another command propmpt and go into the same location `experiment\react-native-uifabric` and run:
```
Comment (dotnetjunky) [inline experiments/tester/README.md]: Update folder path here as well

File: scripts/utils/queryFiles.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    return packageName ? packageJson.name : normalizedPath;
+  } catch {}
+}
+
+function queryPackages(packageNames) {
+  const gitRoot = require('./findGitRoot')();
+  const lernaData = require(gitRoot + '/lerna');
+  const packages = lernaData.packages;
+  const results = [];
+  // "packages": ["packages/*", "scripts", "apps/*", "experiments/*"],
```
Comment (dotnetjunky) [inline scripts/utils/queryFiles.js]: delete this line?

Comment (JasonVMo) [reply scripts/utils/queryFiles.js]: Made it clear that this is informational and not just a commented line of code

**Pull request merged**